pkg/plugins/generic/generic_plugin.go

```diff
@@ -68,8 +69,16 @@ func WithSkipVFConfiguration() Option {
 	}
 }
 
+// WithSkipBridgeConfiguration configures generic_plugin to skip configuration of the managed bridges
+func WithSkipBridgeConfiguration() Option {
```

Comment:
I don't see this one used.

 so at the end we only validate the  `vars.ManageSoftwareBridges`. on the if state here `return vars.ManageSoftwareBridges && !p.skipBridgeConfiguration`
 
also (I know it's not part of this PR) can you please explain me when on prePhase we don't configure the vfs?
https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/f8cc3364c9a16d42a666968dff90dbfbf7219260/cmd/sriov-network-config-daemon/service.go#L209-L210

Reply:
> also (I know it's not part of this PR) can you please explain me when on prePhase we don't configure the vfs?

At the `pre` stage we creatr VFs, but unbind them from a driver to let OS Network manager (e.g. NetworkManager, systemd-network) to apply custom configuration for the PF function (e.g. VF-lag). Such configuration can't be when the PF has VFs that are bound to a driver.

Reply:
The WithSkipBridgeConfiguration function is used in the service command to disable bridge management during the `pre` phase

Reply:
you are right! I miss that sorry